src/test/java/unit/Workspace.java

```diff
@@ -188,6 +188,32 @@ void statusDescribeListReflectUpdate() throws IOException {
     TestCommand.runCommandExpectSuccess("workspace", "delete", "--quiet");
   }
 
+  @Test
+  @DisplayName("update properties in workspace")
+  void updateProperty() throws IOException {
+    TestUser testUser = TestUser.chooseTestUserWithSpendAccess();
+    testUser.login();
+    String initialProperties = "key=value";
+
+    // create a workspace with 1 properties
```

Comment:
Let's do

// Create a workspace with 2 properties: A and B
// Call set-property for properties B and C. (Change what B is set to.)
// Assert properties A, B, and C are correct